self-tests.yml

```diff
@@ -432,6 +437,16 @@ tasks:
             set -o errexit
 
             ./clients/linux_amd64/evergreen service deploy test-endpoints --commit ${revision} --username admin --key abb623665fdbf368a1db980dde6ee0f0
+  - name: js-tests
+    commands:
+      - command: git.get_project
+        type: system
+        params:
+          directory: gopath/src/github.com/evergreen-ci/evergreen
+      - func: run-make
+        vars:
+          target: "js-test"
+          build_env: "PATH=/opt/node/bin:$PATH"
```

Comment:
I'd just add this to the build variant's {{build_env} and then just do it like a normal task.

Reply:
Changed